build.sbt

```diff
@@ -11,15 +11,15 @@ scalaVersion := "2.10.3"
 lazy val root = (project in file(".")).enablePlugins(ShipAutoPlugin)
 
 libraryDependencies ++= Seq(
-  "org.scalatest" %% "scalatest" % "2.0",
-  "org.seleniumhq.selenium" % "selenium-java" % "2.43.1",
+  "org.scalatest" %% "scalatest_2.10" % "2.2.4",
```

Comment:
yeah maybe we should consider doing cross builds!  at the moment we only publish 2.10.  any takers?